internal/nmc/module_nmc_controller.go

```diff
@@ -0,0 +1,142 @@
+package nmc
```

Comment:
Can we rename this file to `internal/nbc/module_nmc.go`? This is just a library and not a controller.

Reply:
fixed